dpe/src/commands/destroy_context.rs

```diff
@@ -57,8 +57,8 @@ impl CommandExecution for DestroyCtxCmd {
             .take(contexts_to_destroy.len())
         {
             if *context_to_destroy {
-                dpe.contexts[idx].destroy();
                 let parent = dpe.contexts[idx].parent_idx as usize;
+                dpe.contexts[idx].destroy();
```

Comment:
Instead of `dpe.contexts[idx]` can we use `context_to_destroy`?

Reply:
context_to_destroy is just a bool, so how would we use it?

Reply:
Oh, never mind, I misread and thought it was the actual context.